programs/uxd/src/instructions/redeem.rs

```diff
@@ -170,6 +159,7 @@ pub(crate) fn handler(ctx: Context<Redeem>, redeemable_amount: u64) -> Result<()
     let identity_depository = ctx.accounts.identity_depository.load()?;
     let mercurial_vault_depository = ctx.accounts.mercurial_vault_depository.load()?;
     let credix_lp_depository = ctx.accounts.credix_lp_depository.load()?;
+    let alloyx_vault_depository = ctx.accounts.alloyx_vault_depository.load()?;
 
     // Compute real outflow limit for this epoch (max of bps/amount options)
```

Comment:
Good point, not sure we need to wrap the whole feature, but i'm going to scope out some of the maths so that we dont leak variables in the function's scope 👌